config/refunds.php

```diff
@@ -2,4 +2,18 @@
 
 return [
 
+    /**
+     * User model
+     */
+    'user_model'    => \App\Models\User::class,
+
+    /**
+     * Payment model
+     */
+    'payment_model' => \App\Models\Payment::class,
+
+    /**
+     * Voucher model
+     */
+    'voucher_model' => \App\Models\Voucher::class,
```

Comment:
We should pin down syntax for how we want to do express these.   I did it slightly different in Disounts
```
    'model_class' => [
        'user' => \App\Models\User::class,
        'order' => \App\Models\Order::class,
        'cart' => \App\Models\Cart::class,
    ],

    'nova_class' => [
        'user' => \App\Nova\User::class,
    ],
```

Don't care which way we go, just prefer it be the same.

Reply:
@pdbreen agreed. I added it to the guidelines file as well.

Reply:
Let's use the syntax that Patrick used in the Discounts package. I think that is cleaner and will be easier for packages that have multiple models.